<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-07T23:23:56 IST" name="Suite" finished-at="2025-08-07T23:24:29 IST" duration-ms="33000">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" name="LoginErrorValidation" class="mavenproject.tests.ErrorValidationsTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="StandAloneTest2.submitOrder(java.util.HashMap)[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" name="submitOrder" class="mavenproject.tests.StandAloneTest2"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2025-08-07T23:23:56 IST" name="Error Validations Test" finished-at="2025-08-07T23:24:24 IST" duration-ms="28553">
      <class name="mavenproject.tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:23:56 IST" name="launchApplication" finished-at="2025-08-07T23:24:04 IST" duration-ms="8322" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:04 IST" name="LoginErrorValidation" finished-at="2025-08-07T23:24:08 IST" duration-ms="3581" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at mavenproject.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:08 IST" name="tearDown" finished-at="2025-08-07T23:24:09 IST" duration-ms="694" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:09 IST" name="launchApplication" finished-at="2025-08-07T23:24:13 IST" duration-ms="4380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:13 IST" name="LoginErrorValidation" finished-at="2025-08-07T23:24:15 IST" duration-ms="1709" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at mavenproject.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:16 IST" name="tearDown" finished-at="2025-08-07T23:24:16 IST" duration-ms="469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:16 IST" name="launchApplication" finished-at="2025-08-07T23:24:20 IST" duration-ms="3967" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:20 IST" name="ProductErrorValidation" finished-at="2025-08-07T23:24:24 IST" duration-ms="3540" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.ErrorValidationsTest@17503f6b]" started-at="2025-08-07T23:24:24 IST" name="tearDown" finished-at="2025-08-07T23:24:24 IST" duration-ms="374" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- mavenproject.tests.ErrorValidationsTest -->
    </test> <!-- Error Validations Test -->
    <test started-at="2025-08-07T23:23:56 IST" name="Submit Order Test" finished-at="2025-08-07T23:24:29 IST" duration-ms="33000">
      <class name="mavenproject.tests.StandAloneTest2">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:23:57 IST" name="launchApplication" finished-at="2025-08-07T23:24:04 IST" duration-ms="7258" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:04 IST" name="submitOrder" data-provider="getData" finished-at="2025-08-07T23:24:12 IST" duration-ms="8086" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Ecom@123, product=ZARA COAT 3, email=ecomuser@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:12 IST" name="tearDown" finished-at="2025-08-07T23:24:13 IST" duration-ms="750" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:13 IST" name="launchApplication" finished-at="2025-08-07T23:24:18 IST" duration-ms="4751" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:18 IST" name="submitOrder" data-provider="getData" finished-at="2025-08-07T23:24:24 IST" duration-ms="5913" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Ecom@321, product=ADIDAS ORIGINAL, email=ecomuser2@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:24 IST" name="tearDown" finished-at="2025-08-07T23:24:24 IST" duration-ms="486" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:24 IST" name="launchApplication" finished-at="2025-08-07T23:24:27 IST" duration-ms="2677" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:27 IST" name="OrderHistoryTest" finished-at="2025-08-07T23:24:28 IST" depends-on-methods="mavenproject.tests.StandAloneTest2.submitOrder" duration-ms="1552" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:mavenproject.tests.StandAloneTest2@38467116]" started-at="2025-08-07T23:24:28 IST" name="tearDown" finished-at="2025-08-07T23:24:29 IST" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- mavenproject.tests.StandAloneTest2 -->
    </test> <!-- Submit Order Test -->
  </suite> <!-- Suite -->
</testng-results>
